criu/memfd.c

```diff
@@ -355,14 +355,6 @@ int memfd_open(struct file_desc *d, u32 *fdflags)
 	close(fd);
 	fd = _fd;
 
-	if (restore_fown(fd, mfe->fown) < 0)
-		goto err;
-
-	if (lseek(fd, mfe->pos, SEEK_SET) < 0) {
-		pr_perror("Can't restore file position of memfd id=%d", mfe->id);
-		goto err;
-	}
-
 	return fd;
 
 err:
```

Comment:
criu/memfd.c: In function ‘memfd_open_fe_fd’:
1342
criu/memfd.c:382:3: error: label ‘err’ used but not defined
1343
  382 |   goto err;
1344
      |   ^~~~
1345
criu/memfd.c: At top level:
1346
cc1: error: unrecognized command line option ‘-Wno-unknown-warning-option’ [-Werror]
1347
cc1: error: unrecognized command line option ‘-Wno-dangling-pointer’ [-Werror]
1348
cc1: all warnings being treated as errors